core/infra/config/peer_service_with_json.cpp

```diff
@@ -124,11 +124,14 @@ size_t PeerServiceConfig::getMaxFaulty() {
 bool PeerServiceConfig::isMyActive() {
     return is_active;
 }
+
 bool PeerServiceConfig::active() {
-    is_active == true;
+    return is_active == true;
```

Comment:
sorry,
`active()` means that myself peer active!
So, `is_active  = true` is correct.